### ISSUE TITLE
Two minor bugs in the initial code

### ISSUE THREAD

**Issue by Owen-Liuyuxuan** [JeffWang987/MOVEDepth#1]
Thanks for the contribution of the code and paper. It is inspiring and helps the community forward.

From the initial version, it seems that two minor bugs are presented in code:

#### Can not import MovedepthOptions:

It seems to be a minor bug when cleaning the code: 

![image](https://user-images.githubusercontent.com/31618608/185879724-32ec1118-117c-4a1b-933e-91018e554309.png)

- probably update [here](https://github.com/JeffWang987/MOVEDepth/blob/fc582699b065121a7bdc4481bdd09d34e82899b8/movedepth/train.py#L5) with

       from movedepth.options import MonodepthOptions

- probably update [here](https://github.com/JeffWang987/MOVEDepth/blob/fc582699b065121a7bdc4481bdd09d34e82899b8/movedepth/train.py#L22) with

       options = MonodepthOptions()

#### Gradient error in newer PyTorch

![image](https://user-images.githubusercontent.com/31618608/185878722-7de1908d-48ce-4de2-9124-583b885d9e44.png)

This is raised because newer pytorch is more restricted to inplace modification that changes the gradients.

![image](https://user-images.githubusercontent.com/31618608/185879518-0cd8b173-48df-4462-812e-e331e83da96d.png)

Probably change [here](https://github.com/JeffWang987/MOVEDepth/blob/fc582699b065121a7bdc4481bdd09d34e82899b8/movedepth/networks/depth_decoder.py#L390) into 

      out = out + x

Then as tested, the code can run at torch==1.11.0

**Comment by z-x-j** [comment]
good
